docs/reference/rosbridge.md

```diff
@@ -95,10 +95,101 @@ ROSBridge.from_ros_boxes(self,
                          ros_detections)
 ```
 Converts a ROS Detection2DArray message with bounding boxes into an OpenDR BoundingBoxList
-        
+
+#### `ROSBridge.from_ros_3Dpose`
+
+```python
+ROSBridge.from_ros_pose(self,
+                        message)
+```
+
+Converts a ROS pose into an OpenDR pose (used for a 3D pose).
+
+Parameters:
+
+- **message**: *geometry_msgs.msg.Pose*  
+  ROS pose to be converted into an OpenDR pose.
+       
+#### `ROSBridge.to_ros_3Dpose`
+
+```python
+ROSBridge.to_ros_pose(self,
+                      opendr_pose)
+```
+Converts an OpenDR pose into a ROS ```geometry_msgs.msg.Pose``` message.
+
+Parameters:
+
+- **opendr_pose**: *engine.target.Pose*  
+  OpenDR pose to be converted to ```geometry_msgs.msg.Pose``` message.
+  
+  #### `ROSBridge.from_ros_3Dpose`
+
+```python
+ROSBridge.from_ros_pose(self,
+                        ros_pose)
+```
+
+Converts a ROS pose into an OpenDR pose (used for a 3D pose).
+
+Parameters:
+
+- **ros_pose**: *geometry_msgs.msg.Pose*  
+  ROS pose to be converted into an OpenDR pose.
+       
+#### `ROSBridge.to_ros_mesh`
+
+```python
+ROSBridge.to_ros_mesh(self,
+                      vertices, faces)
+```
+Converts a triangle mesh consisting of vertices, faces into a ROS ```shape_msgs.msg.Mesh``` message.
+
+Parameters:
+
+- **vertices**: *numpy.ndarray*  
+  Vertices (N) of a triangle mesh.
```

Comment:
```suggestion
  Vertices (Nx3) of a triangle mesh.
```

Reply:
According to the code, this is also Nx3.

Reply:
done